examples/jsf/login/src/main/webapp/users.xhtml

```diff
@@ -44,11 +44,16 @@
 					<td><h:inputText value="#{userManager.newUser.password}"/></td>
 				</tr>
 				<tr>
-					<td></td>
 					<td><h:commandButton action="#{userManager.addUser}" value="Add User" /></td>
 				</tr>
+				<tr>
+					<td><h:link outcome="/home.xhtml">View login page</h:link></td>
+				</tr>
 			</table>
+            <!--TODO FIX THE LOOKS HERE-->
```

Comment:
??

Reply:
Oops, was fiddling with the looks amidst commits and forgot to erase this. Thanks